src/tools/CMakeLists.txt

```diff
@@ -2,7 +2,12 @@ add_subdirectory(batchrun)
 add_subdirectory(finder)
 add_subdirectory(updater)
 add_subdirectory(cleaner)
-add_subdirectory(yby-aggregator)
+
+#yby-aggregator need swap build
+if(BUILD_SWAP)
+    add_subdirectory(yby-aggregator)
+endif()

```

Comment:
yby-agregator's main.cpp has a few
```cpp
#ifdef ANTARES_SWAP_SUPPORT
// ...
#endif
```
So it should work without swap support, right ? Does yby-aggregator depend on libsomething-swap ?

Reply:
Consider adding the `ANTARES_SWAP_SUPPORT`flag instead

Reply:
In CMakeLists.txt, yby-aggregator depends on 
```

set(YBY_AGGREGATOR_LIBS
	libantares-core-swap
	libantares-core-calendar
	yuni-static-core
	libantares-license
	${wxWidgets_LIBRARIES} ${CMAKE_THREADS_LIBS_INIT})
```

Reply:
I suspect that yby aggregator needs a swap support because it may need a lot of memory when running.
The `#ifdef` probably need to be reviewed